technic/machines/LV/compressor.lua

```diff
@@ -5,7 +5,7 @@ minetest.register_alias("compressor", "technic:lv_compressor")
 minetest.register_craft({
 	output = 'technic:lv_compressor',
 	recipe = {
-		{'default:stone',            'basic_materials:motor',          'default:stone'},
+		{'group:stone',            'basic_materials:motor',          'group:stone'},
```

Comment:
Using groups here might cause issues with some mod combinations / configurations, would it be better to be explicit and also add stone to materials?

I guess I'm trying to ask if there's a actual reason to expand recipes to groups, basically not changing it for default ensures that it wont affect backwards compatibility in corner cases.

Reply:
(also same question for other recipes that have been updated to group: ingredients)

Reply:
changed to `group:cobble` and `group:stone` in all files